docs/integration/api_exercise_5.md

```diff
@@ -12,50 +12,51 @@ In previous exercises, we’ve shown that you can use the Assemblyline client or
 Send files to Assemblyline through Ingest/Submit API using cURL
 ```
 
-## APIs Involved
-### Web
-`POST /api/v4/submit/`
-
-`POST /api/v4/ingest/`
-
-### cURL Cheatsheet
+## REST APIs Involved
 ```
-# CURL cheat sheet
-#  to pass headers: -H 'key: value'
-#  to set the request type: -X GET
-#  to stop cert validation: -k
-#  to add a multipart form data:
-#        -F 'name=data'
-#    OR  -F 'name=@path_to_file'
-
-# Ingest / Submit API cheatsheet
-#   * JSON parameters to the submission are passed inside a multipart object named 'json'
-#   * The file binary is passed inside a multipart object named 'bin'
-
-# ** Tip: you can pipe the curl output to json_pp so you can actually read it
+POST /api/v4/submit/
+POST /api/v4/ingest/
 ```
 
+??? info "cURL Cheatsheet"
+    ```
+    to pass headers: -H 'key: value'
+    to set the request type: -X GET
+    to stop cert validation: -k
+    to add a multipart form data:
+          -F 'name=data'
+      OR  -F 'name=@path_to_file'
+
+    # Ingest / Submit API cheatsheet
+     * JSON parameters to the submission are passed inside a multipart object named 'json'
+     * The file binary is passed inside a multipart object named 'bin'
+
+    # ** Tip: you can pipe the curl output to json_pp so you can actually read it
+    ```
+
 ## Solution
 
-```bash
-# Send a file for asynchronous processing using CURL
-# ** API to use: /api/v4/ingest/ (POST)
-echo "Send to ingest API:"
-curl -s -k -X POST https://$AL_HOST/api/v4/ingest/ \
-    -H "x-user: ${AL_USER}" \
-    -H "x-apikey: ${AL_APIKEY}" \
-    -H 'accept: application/json' \
-    -F 'json={"params": {"description": "My CURL test"}, "metadata": {"any_key": "any_value"}}' \
-    -F 'bin=@myfile.txt' | json_pp
-
-# Send a file for live processing using CURL
-# ** API to use: /api/v4/submit/
-echo ""
-echo "Send to submit API:"
-curl -s -k -X POST https://$AL_HOST/api/v4/submit/ \
-    -H "x-user: ${AL_USER}" \
-    -H "x-apikey: ${AL_APIKEY}" \
-    -H 'accept: application/json' \
-    -F 'json={"params": {"description": "My CURL test"}, "metadata": {"any_key": "any_value"}}' \
-    -F 'bin=@myfile.txt' | json_pp
-```
+=== "Using Submit API"
+    ```bash
+    # Send a file for live processing using CURL
+    # ** API to use: /api/v4/submit/
```

Comment:
I think this was missing from the previous solution, to reflect what is given using Ingest:
```suggestion
    # ** API to use: /api/v4/submit/ (POST)
```